pipelines/api-deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: api-pipeline
+name: api-deploy
```

Comment:
How would the deploy pipeline know that there is something to deploy? Do we need to add a dependency on the build pipeline or something?

Reply:
It would auto run based on the defined triggers.  Ex) Runs when new code is checked into master and passes the path include checks.

Reply:
That part I understood from the paths below, but my question is about cross pipeline dependency, how would that take place? Before we used a condition to force a dependency between stages but now I dont see how that is taking place.

Reply:
What cross pipeline dependency?

Reply:
@alkofahi not sure what you're asking.  This PR doesn't change any existing features, code are still deploy once changes has been merge to master